rts/System/FileSystem/Archives/BufferedArchive.cpp

```diff
@@ -47,12 +39,19 @@ bool CBufferedArchive::GetFile(unsigned int fid, std::vector<std::uint8_t>& buff
 
 	FileBuffer& fb = fileCache.at(fid);
 
+	fb.numAccessed++;
 	if (!fb.populated) {
-		fb.exists = ((ret = GetFileImpl(fid, fb.data)) == 1);
-		fb.populated = true;
-
-		cacheSize += fb.data.size();
-		fileCount += fb.exists;
+		if (fb.numAccessed > 1) {
+			fb.exists = ((ret = GetFileImpl(fid, fb.data)) == 1);
+			fb.populated = true;
+
+			cacheSize += fb.data.size();
+			fileCount += fb.exists;
+		}
+		else {
```

Comment:
Consider an explanation
```suggestion
		}
		else { // most files are only accessed once, don't bother with those
```